content/src/content/jcr_root/apps/core/wcm/sandbox/components/tabs/v1/tabs/clientlibs/site/js/tabs.js

```diff
@@ -86,6 +90,17 @@
             }
         }
 
+        function getActiveIndex(tabs) {
```

Comment:
Could you please add the jsdocs?

Reply:
Done in https://github.com/Adobe-Marketing-Cloud/aem-core-wcm-components/pull/317/commits/bdd641e41c0331b00636b7113b89f0432fc0c225